src/library/Uno.Material/Styles/Application/AnimationConstants.xaml

```diff
@@ -7,4 +7,9 @@
 	<CubicEase x:Key="Material_EaseOutFunction"
 			   EasingMode="EaseOut" />
 
+	<Duration x:Key="Material_AnimationDuration">0:0:0.25</Duration>
+	<Duration x:Key="Material_AnimationDuration_Short">0:0:0.15</Duration>
```

Comment:
As @Xiaoy312 mentioned. I think we should have a clear "usage" of the MaterialAnimationDuration properties, depending on where they are used.
For example, we could say all pressed and pointer over animations are the same. We must have a reason for having different values, and developers could try to override those resources to tweak it to their liking so they must know what they mean.